Misc/NEWS.d/next/Library/2019-11-21-16-30-00.bpo-38863.RkdTjf.rst

```diff
@@ -0,0 +1,2 @@
+Improve :func:`is_cgi` function in http.server module, which enables processing
```

Comment:
````
:mod:`http.server`
````

Reply:
I appreciate for your valuable advice. I will get back with the change soon.